peps/pep-0007.rst

```diff
@@ -135,6 +135,21 @@ Code lay-out
           return 0; /* "Forgive" adding a __dict__ only */
       }
 
+* Vertically align line continuation characters in multi-line macros.
+
+* Macros intended to be used as a statement should use the
+  ``do {... } while(0)`` macro idiom,
+  without a final semicolon.
+  Example::
+
+      #define ADD_INT_MACRO(MOD, INT) do {                          \
```

Comment:
Sorry, latecomer here. I'd also prefer if the example had `do {` on its own line, even if we're not requiring a style. This is consistent with our bracing rules for e.g. functions. Plus, it's easy to overlook the "zero-th" line of a multi-line macro.

```suggestion
      #define ADD_INT_MACRO(MOD, INT)                               \
      do {                                                          \
```

Reply:
Ok, both you and Victor have suggested this; I'll change the example :)

Reply:
Addressed in 31be7f224cec8d6b09a5aa20d913c87a9384e506

Reply:
Well, I asked to add one indentation level. At top level, it avoids to be confused with other code.

Reply:
I'm fine that; I think the original indent in this PR only makes sense if `do {` is included in the first (or zero'th) line.

Reply:
Addressed in ed5ee6d665490da257e26d63aeaf164acc863cc8

Reply:
Perfect, thank you. Sometimes putting do { on the define line makes the code more readable, but it's more the exception than the rule.